src/platform/nm-linux-platform.c

```diff
@@ -3068,6 +3316,10 @@ typedef struct {
 
 		gint is_handling;
 	} delayed_action;
+
+	GHashTable *wifi_data;
```

Comment:
unused field

Reply:
That's just a mistake while rebasing - I can drop and update this PR.